docs/tutorials/connecting-to-the-internet-via-usb.md

```diff
@@ -137,447 +108,223 @@ If you are using an older version, please upgrade.
+    3.  Wait a minute for Windows install the driver. It will detect the EV3
+        as a *Remote NDIS Compatible Device*.
 
-        23. Now, open up the *Network & Internet* settings either using the notification
-            tray icon or the Start menu. Select *Ethernet* on the left, then click
-            on *Change adapter options*.
+        {% include screenshot.html source="/images/windows/10/devices-and-printers-rndis.png" %}
 
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/settings-network-and-internet-ethernet.png" %}
+        <span class="glyphicon glyphicon-info-sign" aria-hidden="true"></span>
+        PROTIP: You can get an icon that looks like an actual EV3 by installing
+        [Programmable Brick Icons](https://github.com/ev3dev/PBrickIcons).
+        Follow the link for more information.
+        {% include screenshot.html source="https://raw.githubusercontent.com/ev3dev/PBrickIcons/master/demo1.png" %}
+        {:class="alert alert-info"}
 
-        24. There will be a connection that says *USB Ethernet/RNDIS Gadget*.
+    4.  Right-click the *Remote NDIS Compatible Device* and select *Network Settings*.
+        This takes you to the *Network and Sharing Center*.
 
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/network-connections.png" %}
+        {% include screenshot.html source="/images/windows/10/devices-and-printers-context-menu-network-settings.png" %}
 
-        25. This is the EV3. Let's rename it so it is easy to identify. Right-click the
-            *USB Ethernet/RNDIS* adapter and select *Rename*. Call it EV3 or whatever
-            you like.
+    5.  In the *Network and Sharing Center*, click on *Change adapter settings*.
 
-            {% include screenshot.html source="/images/screenshots/windows/10/rndis/network-connections-rename.png" %}
+        {% include screenshot.html source="/images/windows/10/network-and-sharing-center-change-adapter-settings.png" %}
 
-        26. If you don't need to access the Internet from your EV3 over this
-            connection, manually configure the IP address to `192.168.137.1`. Note:
-            you won't be able to update packages without an Internet connection.
+    6.  There will be a connection that says *Remote NDIS Compatible Device* (1).
+        This is the EV3. Let's rename it so it is easy to identify. Right-click
+        the adapter and select *Rename* (2). Call it EV3 or whatever you like.
 
-            TODO: Need detailed description with screenshots.
+        {% include screenshot.html source="/images/windows/10/network-connections-context-menu-rename.png" %}
 
-            Then skip to step 18.
+    7.  To share our Internet connection with the EV3, Double-click the other
+        network connection (**not** the one we just renamed). This will show the
```

Comment:
It doesn't look like it is explicitly mentioned that you want to select your incoming internet connection -- it's only mentioned as a side note in the next step. I'd recommend that you say what they are actually looking for (the incoming connection) instead of just "the other one".